detox/local-cli/detox-test.js

```diff
@@ -38,6 +38,9 @@ switch (program.runner) {
   case 'mocha':
     command = `node_modules/.bin/${program.runner} ${testFolder} --opts ${testFolder}/${program.runnerConfig} ${configuration} ${loglevel} ${cleanup} ${reuse} ${debugSynchronization} ${artifactsLocation}`;
     break;
+  case 'jest':
```

Comment:
What about passing detox command line args? Will those work?

Reply:
We can think of a way doing this. Although, in this version I wasn't aiming to provide this. If this is mandatory, we can discuss the approach

Reply:
I think it is, the feature is not 100% usable without it.

Reply:
`detox test --configuration ios.sim.release`
`detox test --loglevel verbose`
`detox test --debug-synchronization`

Reply:
Which flags are important? 
In the previous version there were no recommendations regarding flags so kept it out of scope. However, I'm fine adding important flags to make jest support even better 💪 .

Reply:
> detox test --configuration ios.sim.release
detox test --loglevel verbose
detox test --debug-synchronization

Will add these ones ASAP 💪

Reply:
We need all of them ;)

Reply:
I mean, let's try to add a way to make the detox cli a passthrough, and not manually add each command when we add it to Detox itself.
I know it's not how we do it with our Mocha support, but we need to up our game and create better code even in the cli tool.

Reply:
@rotemmiz what is the goal?
- tests can access CLI arguments
or
- detox can access CLI arguments
---
If we talk about the second one, then it already works like this (commander parses arguments before passing them to test runner)

Reply:
You're right, but we execute a child process and these argument are not being passed into it.
`cp.execSync(command, {stdio: 'inherit'});` 
We probably just need to pass them on...

Reply:
Oh, now I see. Indeed, we just need to pass arguments to the child process. Then there will be no point in passing all these flags in the command itself

Reply:
Seems like `jest` doesn't allow custom CLI parameters. The only approach I see is to use environment vars